.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -43,7 +43,7 @@ body:
       description: Where should we change this
       options:
         - Homepage
-        - Componenten overzicht (zoek pagina)
+        - Componenten overzicht (zoekpagina)
         - Component detail pagina
         - Organisatie detail pagina
         - Documentatie pagina's
```

Comment:
This list still has some merging of languages (e.g. "Homepage" and "pagina" in the same list).

Reply:
Yeahh, I found this one confusing as wel. The overal mix of Dutch and English on site and repo doesn't sit well.